.readthedocs.yml

```diff
@@ -1,8 +1,9 @@
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 version: 2
 sphinx:
-  configuration: docs/conf.py
+  configuration: components/google-cloud/docs/source/conf.py
```

Comment:
We should only change the root file in release branches, otherwise this will break kfp sdk documentation since both SDKs share the same repo.

Reply:
done.